mkdocs.yml

```diff
@@ -177,6 +177,7 @@ nav:
       - Privacy:
         - Create a privacy-enabled network: Tutorials/Privacy/Configuring-Privacy.md
         - Configure a multi-tenant network: Tutorials/Privacy/Configuring-Multi-Tenancy.md
+        - Private transaction performance best practices: Tutorials/Privacy/Private-Transaction-Performance-Best-Practice.md
```

Comment:
does this belong under tutorials?

Reply:
what about under Concepts/Privacy

Reply:
Good question. Would you rather have it under `How to -> Use privacy features`?